typescript/sdk/src/token/EvmERC20WarpModule.ts

```diff
@@ -182,12 +246,24 @@ export class EvmERC20WarpModule extends HyperlaneModule<
     const deployer = new HypERC20Deployer(multiProvider);
     const deployedContracts = await deployer.deployContracts(chainName, config);
 
-    return new EvmERC20WarpModule(multiProvider, {
+    const warpModule = new EvmERC20WarpModule(multiProvider, {
       addresses: {
         deployedTokenRoute: deployedContracts[config.type].address,
       },
       chain,
       config,
     });
+
+    // Enroll Remote Routers
+    if (config.remoteRouters && !isObjEmpty(config.remoteRouters)) {
+      const enrollRemoteTx = await warpModule.updateRemoteRouters(
```

Comment:
even though it's always a single tx, this should still be named `enrollRemoteTxs`

Reply:
Fixed